### ISSUE TITLE
Ontology error when I gave it a zip code in Enroll

### ISSUE THREAD

**Issue by candysidner** [always-on/always#279]
I chose a zip code at random in hopes that it wasn't one--to test the response to bad zipcodes (which is in enroll).  I got an error in Enroll, see:
https://www.dropbox.com/s/04du8tj469bfm1e/Ontologyerror.test

**Comment by charlesrich** [comment]
Could you please also put the user model (name starts with BAD) in DropBox also and link in issue.

Tx, -Chuck

**Comment by candysidner** [comment]
the BAD user model is: https://www.dropbox.com/s/6x3eybgj8lxuixe/BAD.User.2014-02-10_10_14_48.owl

**Comment by mbehrooz** [comment]
I did two kinds of tests on this. Firstly, I entered two entries with the same Zips which sounded like it could be the bug, at least judging by the Exception name (Duplicate Keys not allowed. Duplicate for 'java.io.File:edu.wpi.always.user.UserModel$UserOntologyLocation') But I saw no problem with that.

Secondly, I of course entered the particular Zip code in the log, and that seemed to work fine too (it is a valid one).

So I am wondering if this is not related to the Zip code at all. I will think more on this, but just as an update to limit the cases.  

Thanks, 
Morteza
